### ISSUE TITLE
Misleading documentation/default values for window layout

### ISSUE THREAD

**Issue by marcusfuchs** [RWTH-EBC/TEASER#559]
## What is the problem?

The Office building archetype has a parameter `window_layout`. The documentation gives the following information:

```
window_layout : int
    Structure of the window facade type, default is 1, which is
    representative for a punctuated facade.
        1: punctuated facade (individual windows)
        2: banner facade (continuous windows)
        3: full glazing
```

This leads me to believe that when creating an office building like this, we get a puncuated facade:

```Python
prj.add_non_residential(
        method="bmvbs",
        usage="office",
        name="BuildingWithPunctuatedFacadeExpected",
        year_of_construction=year,
        number_of_floors=2,
        height_of_floors=3.5,
        net_leased_area=1000.0,
    )
```

Yet, such a building yields a different heating demand compared to 

```Python
prj.add_non_residential(
        method="bmvbs",
        usage="office",
        name="BuildingWithPunctuatedFacadeActual",
        year_of_construction=year,
        number_of_floors=2,
        height_of_floors=3.5,
        net_leased_area=1000.0,
        window_layout=1,
    )
```

This seems to be caused by the office archetype, which actually defines not only 3 but 4 options for the window layout, with `0` as default instead of `1`:

```Python
        if self.window_layout == 0:
            self.corr_factor_wall = 1.0
            self.corr_factor_win = 1.0
        elif self.window_layout == 1:
            self.corr_factor_wall = 0.75
            self.corr_factor_win = 0.25
        elif self.window_layout == 2:
            self.corr_factor_wall = 0.5
            self.corr_factor_win = 0.5
        elif self.window_layout == 3:
            self.corr_factor_wall = 0.1
            self.corr_factor_win = 0.9
        else:
            raise ValueError("window_layout value has to be between 0 - 3")
```

## Why do we want to solve it?

I think correcting this will help users get the results they expect.

## How do we want to solve it?

I am not sure what `window_layout = 0` represents in the building. So I would suggest updating the documentation and/or the code.

**Comment by marcusfuchs** [comment]
@MichaMans @PRemmen if you could share your opinion on how to proceed with this I could offer to implement a fix.

**Comment by PRemmen** [comment]
@marcusfuchs sorry for the late reply. From my feeling the code

```
if self.window_layout == 0:
            self.corr_factor_wall = 1.0
            self.corr_factor_win = 1.0
```

is wrong. I do not remember the another facade category but 

1: punctuated facade (individual windows)
2: banner facade (continuous windows)
3: full glazing

But I'll look it up in the old code

**Comment by marcusfuchs** [comment]
Thanks! When you have checked the old code, I would suggest that I remove everything related to `window_layout = 0` and update the documentation and default values. For me it's no problem, but I expect that everyone who used the archetype without specifying a window layout explicitly will get different results after this change.

**Comment by mlauster** [comment]
As far as I can recall, 0 is just a default value, that is not related to any specificly named layout. More a kind of statistical mean value. 
In this sense, I would leave the code as it is and simply update the documentation.

**Comment by marcusfuchs** [comment]
@mlauster Thanks for sharing your input! I have no objections to leaving the `0` option in the code. To update the documentation, I still have some difficulties understanding what `0` represents. If `1` - `3` cover everything from individual windows to all-window facades, I am not sure how to imagine a building with `window_layout = 0`. I suppose it does not mean "no windows", but can it be something like "less windows than individual windows"?

**Comment by marcusfuchs** [comment]
I would also expect that `0` can't be no windows. I would interpret the code to use both correction factors at value 1 so that no correction is applied to some base value. So option `0` would output this base value. If this base case had no windows, all correction factors for options `1` - `3` would multiply `0 * factor`, so no option could have any windows. 

I am just not sure if exposing option `0` to users (especially as a default value) gives a meaningful model or whether this base case is originally intended to be only used after having been corrected in options `1` - `3`. At least personally as a user I would not expect that there could be an option with less window area than option `1`.

**Comment by mlauster** [comment]
That's the relevant code snippet
    
    if not self.window_layout == 0:
     self._est_outer_wall_area = self._est_facade_area * \ self.corr_factor_wall
     self._est_win_area = self._est_facade_area * self.corr_factor_win
    else:
     pass
    
So no correction is applied for case 0. In conclusion, it's a generic facade representing a statistical mean value of window area.

**Comment by PRemmen** [comment]
Nice discussion, thanks for the input @mlauster! In that case a simple correction of the documentation would be sufficient?

**Comment by mlauster** [comment]
Yep, that's the solution!

**Comment by marcusfuchs** [comment]
Yes, also fine with me - I just didn't get around to create a PR (may still take a bit of time...).

**Comment by marcusfuchs** [comment]
From your classification I take it that this should be merged against `development` rather than `master`. I'll send a PR.

**Comment by mlauster** [comment]
Closed by #565
